src/index.ts

```diff
@@ -40,6 +40,38 @@ const createGenerator: CodegenGeneratorConstructor = (config, context) => {
 			return !!value.mediaType.mimeType.match('\\bjson$')
 		})
 
+		hbs.registerHelper('hasOperationSupportingMultipart', function(value: CodegenOperation | CodegenOperation[]): boolean {
+			if (isCodegenOperation(value)) {
+				return value.requestBody?.defaultContent.mediaType.mimeType === 'multipart/form-data'
+			}
+
+			for (const op of value) {
+				if (op.requestBody?.defaultContent.mediaType.mimeType === 'multipart/form-data') return true
+			}
+			return false
+		})
+
+		hbs.registerHelper('isSchemaSupportsMultipart', function(value: CodegenSchema): boolean {
+			return !!value.nativeType.toString().match('Api\\.(\\w+)\\.MultipartFormData(\\.\\w*)?')
+		})
+
+		hbs.registerHelper('isFileSchema', function(value: CodegenProperty): boolean {
```

Comment:
I think this should probably be added to the Handlebars generator itself, as file is a core type.

Reply:
I think I could remove this instead cos it's only used in one place and it functions the same as passing `schema` to `isFile`?